src/components/login/ContactUs.vue

```diff
@@ -1,120 +1,141 @@
-          <input
-              id="email"
-              type="email"
-              v-model="formData.email"
-              required
-              placeholder="Email"
-          />
-        </div>
-        <div class="input-group">
-          <label for="phoneNumber">Phone Number</label>
-          <div class="phone-input">
-            <select v-model="formData.countryCode">
-              <option value="+1">+1</option>
-              <option value="+91">+91</option>
-            </select>
+        <!-- Email and Phone Number -->
+        <div class="input-row">
+          <div class="input-group">
+            <label for="email">Email</label>
             <input
-                id="phoneNumber"
-                type="tel"
-                v-model="formData.phoneNumber"
+                id="email"
+                type="email"
+                v-model="formData.email"
                 required
-                placeholder="Phone Number"
+                placeholder="Email"
+                aria-label="Email"
             />
           </div>
+          <div class="input-group">
+            <label for="phoneNumber">Phone Number</label>
+            <div class="phone-input">
+              <select v-model="formData.countryCode" aria-label="Country Code">
+                <option value="+1">+1</option>
+                <option value="+91">+91</option>
+              </select>
+              <input
+                  id="phoneNumber"
+                  type="tel"
+                  v-model="formData.phoneNumber"
+                  required
+                  placeholder="Phone Number"
+                  aria-label="Phone Number"
+              />
+            </div>
+          </div>
         </div>
-      </div>
-
-      <div class="input-row">
-        <div class="input-group">
-          <label for="interest">What are you interested in?</label>
-          <select id="interest" v-model="formData.interest" required>
-            <option value="" disabled>Select</option>
-            <option value="product">Product</option>
-            <option value="service">Service</option>
-            <option value="other">Other</option>
-          </select>
+
+        <!-- Interest and Country/Region -->
+        <div class="input-row">
+          <div class="input-group">
+            <label for="interest">What are you interested in?</label>
+            <select id="interest" v-model="formData.interest" required aria-label="Interest">
+              <option value="" disabled>Select</option>
+              <option value="product">Product</option>
+              <option value="service">Service</option>
+              <option value="other">Other</option>
+            </select>
+          </div>
+          <div class="input-group">
+            <label for="country">Country/Region</label>
+            <select id="country" v-model="formData.country" required aria-label="Country">
+              <option value="India">India</option>
```

Comment:
<option value="India">India</option>

here option value and what to display inside option should come from array of objects and we can loop it.
Check if we can use v for like https://github.com/Admin-Portal-VDS/Admin-Portal/commit/4c882022cec98fe11aef98c2e660265c8b11f3de#diff-850bd6da8305b1dcdc873c1f252fec006d5b17456b00868a92d3a8bde3fb2cecR10 here

```

const countryDropdownOptions = [
    { value: "India", displayText: "India" },
    { value: "UK", displayText: "United Kingdom" },
];

```

Reply:
same for other dropdowns like https://github.com/Admin-Portal-VDS/Admin-Portal/pull/22/files#diff-3db29592d1406a78e91361d34bbf832fbd992778b4a3df8cbbb2049958826914R49-R54